Makefile

```diff
@@ -34,7 +34,7 @@ lint:
 	GOGC=25 golangci-lint run --timeout=3m
 
 run:
-	GOGC=25 go run main.go
+	GOGC=25 go run main.go --v=2
```

Comment:
Why do we turn up the verbosity here?

Reply:
When we run `make run` without `--v=2`, collector hides the groups/resources it finds. This was mostly useful when testing allow/deny list. However, I'm not sure if we want to keep this change so those logs are always visible. Thoughts? @jlpadilla

Reply:
This line only affects development.  It's ok to develop with a higher verbosity if that helps. I have set a similar default in other projects.

This doesn't affect production, but just FYI, in prod we should try to reduce the default logging to only a summary, so that problems and potential problems stand out.